src/test/browser/fixtures/esbuild-browsers-shims.mjs

```diff
@@ -1,8 +1,6 @@
-import * as bufferModule from 'buffer-es6'
-import * as processModule from 'process-es6'
+import * as processModule from 'process'
 
 export const process = processModule
```

Comment:
I'm a little confused here.
Can't you also include `process` in the headers requires as you did for `buffer` so that also that can be removed?

Reply:
This is required by `tape` or a related package and can't be patched in the code.